### ISSUE TITLE
Output Directory Issues on Windows

### ISSUE THREAD

**Issue by tylocook** [facebookincubator/FBX2glTF#109]
While running the Windows binary, I've run into some odd behavior surrounding the -o flag.

In my case, I am running the commandline as **Administrator** for sanity's sake. I also made sure the delete any folders/files created between running commands, so consider all **output folders** as non-existent before running the tool.

Here are the calls to the binary and results:

-o path with **backslashes** on my C:
```
FBX2glTF-windows-x64.exe -o "C:\u000790410058_hr" -i "E:\EDrive\Products_10k\u000790410058\u000790410058_hr.fbx"

ERROR:: Couldn't open file for writing: C:\u000790410058_hr_out/u000790410058_hr.gltf
```

-o path with **forwardslashes** on my C:
```
FBX2glTF-windows-x64.exe -o "C:/u000790410058_hr" -i "E:\EDrive\Products_10k\u000790410058\u000790410058_hr.fbx"

Warning: animation 'Take 001' has zero channels. Skipping.
Wrote 4261 bytes of glTF to C:/u000790410058_hr_out/u000790410058_hr.gltf.
Wrote 9864 bytes of binary data to C:/u000790410058_hr_out/buffer.bin.
```

I also have a D drive that mirrors the same behavior as the C drive above.

-o path with **backslashes** on my E:
```
FBX2glTF-windows-x64.exe -o "E:\u000790410058_hr" -i "E:\EDrive\Products_10k\u000790410058\u000790410058_hr.fbx"  

ERROR: Failed to create folder: E:\u000790410058_hr_out/'
```

-o path with **forwardslashes** on my E:
```
FBX2glTF-windows-x64.exe -o "E:/u000790410058_hr" -i "E:\EDrive\Products_10k\u000790410058\u000790410058_hr.fbx"

ERROR: Failed to create folder: E:/u000790410058_hr_out/'
```

In summary, for some reason, this program has no problem creating an output folder on my C drive, but cannot create the files inside of that output folder properly _when using backslashes_. This problem does not exist when using forwardslashes.

With my E drive, however, it's unable to create an output folder no matter which style of slash I use. This one is truly a mystery to me.

As a side note, attempting to escape the backslash by using \\\ did not help.

**Comment by zellski** [comment]
Thank you for the detailed bug report. I have to be upfront about my near-complete unfamiliarity with Windows-specific programming. I can't usefully guess how D: and E: differ in your system to make for such different behaviour...

That said, I can at least investigate and see if anything obvious falls out. And if you happen to know any strong Windows developers that are looking to help out with an open source project... :-)

**Comment by zellski** [comment]
@tylocook Are you by any chance able to give this a whirl, with a recent pull & build from master? There's been quite a few bug fixes in the file/path handling.

**Comment by zellski** [comment]
This should've been all fixed when we moved to boost::filesystem.

**Comment by harrypatrick442** [comment]
I dont even get a file output. I get a couple of warnings but no errors. This is all but useless. Sigh.
